spokestack/extensions/webrtc/ans.pyx

```diff
@@ -0,0 +1,59 @@
+"""
+This module contains the AutomaticNoiseSuppression class implemented in Cython
+"""
+# distutils: sources = filter_audio/ns/noise_suppression_x.c
+# distutils: include_dirs = filter_audio/ns/include/
+cimport cans
+cimport numpy as np
+
+
+np.import_array()
+
+cdef class AutomaticNoiseSuppression:
+    cdef cans.NsxHandle *_ans
+    cdef int _sample_rate
```

Comment:
`_sample_rate` and `_policy` are not used outside the constructor, so they don't need to be stored